content/english/supply chain analytics/1_Introduction.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Introduction to Supply Chain"
```

Comment:
Please rename this folder to supply-chain-analytics

Although having 1_, 2_ makes it easier to order the pages in the file editor, these prefixes also show up in the URL of the website, which is not ideal... I would remove them.

e.g., rename this